installing_on_aws/index.md

```diff
@@ -7,6 +7,8 @@
 
 {% include configuration.md %}
 
+**Note:** When configuring messaging, please refer to https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-naming.html on hostname guidelines i.e. inclusion of `.ec2.internal` or `.compute.internal` in the hostname
```

Comment:
On AWS the only format that works is the Private IP DNS name (eg ip-172-31-1-251.us-west-1.compute.internal) which can be obtained by running hostname -f

Reply:
Thanks, addressed in a follow-up https://github.com/ManageIQ/manageiq-documentation/pull/1778